ui/page/governance/proposals_page.go

```diff
@@ -363,26 +366,30 @@ func (pg *ProposalsPage) layoutStartSyncSection(gtx C) D {
 }
 
 func (pg *ProposalsPage) listenForSyncNotifications() {
-	go func() {
-		for {
-			var notification interface{}
-
-			select {
-			case notification = <-pg.Receiver.NotificationsUpdate:
-			case <-pg.ctx.Done():
-				return
-			}
 
-			switch n := notification.(type) {
-			case wallet.Proposal:
-				if n.ProposalStatus == wallet.Synced {
-					pg.syncCompleted = true
-					pg.isSyncing = false
+	if pg.proposalNotification {
+		go func() {
+			for {
+				var notification interface{}
+
+				select {
+				case notification = <-pg.Receiver.NotificationsUpdate:
+				case <-pg.ctx.Done():
+					return
+				}
+
+				switch n := notification.(type) {
+				case wallet.Proposal:
+					if n.ProposalStatus == wallet.Synced {
+						pg.syncCompleted = true
+						pg.isSyncing = false
 
-					pg.fetchProposals()
-					pg.RefreshWindow()
+						pg.fetchProposals()
+						pg.RefreshWindow()
+					}
```

Comment:
I dont agree with this as this will prevent the pages from being updated after sync is completed.

The idea of this PR is to prevent the notification that shows up on your computer screen when a new notification is added.

You should be working from here https://github.com/planetdecred/godcr/blob/master/ui/page/listeners.go

https://github.com/planetdecred/godcr/blob/4c5ba0a23f789cfad9c389295bbdfe6eb52e610f/ui/page/listeners.go#L181